zh/option/series/heatmap.md

```diff
@@ -121,6 +121,10 @@ option = {
 
 样式设置，在直角坐标系([coordinateSystem](~series-heatmap.coordinateSystem): 'cartesian2d')上有效。
 
+{{ use: partial-border-radius(
```

Comment:
Put the `partial-border-radius` beind more common used `partial-item-style`

Reply:
Copy that.